src/go/rdctl/cmd/root.go

```diff
@@ -95,67 +110,75 @@ func getRequestObject(method string, command string) (*http.Request, error) {
   return req, nil
 }
 
-func doRestOfRequest(req *http.Request) error {
-  client := http.Client{}
-  response, err := client.Do(req)
-  if err != nil {
-    return err
-  }
-  if response.StatusCode < 200 || response.StatusCode >= 300 {
-    switch (response.StatusCode) {
-    case 401:
-      return fmt.Errorf("user/password not accepted")
-    case 500:
-      return fmt.Errorf("server-side problem: please consult the server logs for more information.")
-    }
-    return fmt.Errorf("%s", response.Status)
-  }
 
-  defer response.Body.Close()
+func doRestOfRequest(req *http.Request) ([]byte, error) {
```

Comment:
Eventually the implementation logic, like request processing, should move from `cmd/root.go` to `pkg/request` or something. Not in this PR, but later refactoring.

Reply:
Sure.